sampler.go

```diff
@@ -511,6 +511,7 @@ func (s *RemotelyControlledSampler) updateSampler() {
 	res, err := s.manager.GetSamplingStrategy(s.serviceName)
 	if err != nil {
 		s.metrics.SamplerQueryFailure.Inc(1)
+		s.logger.Infof("Unable to query sampling strategy. Got error: %v", err)
```

Comment:
simplify the message: `s.logger.Infof("Unable to query sampling strategy: %v", err)`

can you add a unit test?

Reply:
e.g. https://github.com/jaegertracing/jaeger-client-go/blob/f7e0d4744fa6d5287c53b8ac8d4f83089ce07ce8/reporter_test.go#L97

Reply:
preferably use `string.Contains()`

Reply:
Done :)